assets/js/scripts.js

```diff
@@ -430,6 +434,15 @@ L.LayerGroup.include({
   },
 });
 
+const LaIcon = L.Icon.extend({

```

Comment:
Better practice is to add next property to L.Icon object instead of creating new variable. Like this:
```javascript
L.Icon.TimedData = L.Icon.extend({
...
```

Reply:
and the code be moved to BaseMap.init, as you wished it.